src/panels/config/hardware/dialog-hardware-available.ts

```diff
@@ -171,12 +171,19 @@ class DialogHardwareAvailable extends LitElement implements HassDialog {
         ha-icon-button {
           position: absolute;
           right: 16px;
+          inset-inline-end: 16px;
+          inset-inline-start: initial;
           top: 10px;
           text-decoration: none;
           color: var(--primary-text-color);
         }
         h2 {
-          margin: 18px 42px 0 18px;
+          margin-top: 18px;
+          margin-bottom: 0;
+          margin-right: 42px;
+          margin-left: 18px;
```

Comment:
No need to write this out?

Reply:
I thought it would be more explicit. Fixed.